src/Behavioral.Automation/Bindings/ClickBinding.cs

```diff
@@ -27,10 +38,18 @@ public void ClickTwice([NotNull] IWebElementWrapper element)
         public void ClickThreeTimes([NotNull] IWebElementWrapper element)
         {
             element.Click();
+            _debugBinding.Wait(1);
+            _presenceBinding.CheckElementShown(element, new AssertionBehavior(AssertionType.Continuous, false));
```

Comment:
Do we really need this check? Element is checked for being enabled in the wrapper before click

Reply:
removed reduntant check